### ISSUE TITLE
Empty description issue on iOS

### ISSUE THREAD

**Issue by taqueci** [taqueci/redmine_wysiwyg_editor#145]
The issue #141 has not been resolved yet.

> We experienced a bug for iOS (iPhone)
> When we update an issue or a project from a mobile phone (case of textil mode activated) the field description will be deleted
> Because the function _changeMode is called in init_instance_callback
> Since the function _changeMode fire the event blur that call _setTextContent
> In this case self._editor.getContent() is empty

**Comment by taqueci** [comment]
The issue is no longer found in my environment with iPhones:
- iPhone 12 mini (iOS 13.5.1, Safari 604.1)
- iPhone SE 2nd Gen (iOS 13.5.1, Safari 604.1)
- iPhone 7 (iOS 13.5.1, Safari 604.1)
- iPhone 6 (iOS 12.5.5, Safari 604.1)
- iPhone 5 (iOS 10.3.4, Safari 602.1)

```
Environment:
  Redmine version                4.2.4.stable
  Ruby version                   2.7.5-p203 (2021-11-24) [x86_64-linux]
  Rails version                  5.2.6.2
  Environment                    production
  Database adapter               PostgreSQL
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
SCM:
  Subversion                     1.14.1
  Mercurial                      5.6.1
  Bazaar                         3.1.0
  Git                            2.30.2
  Filesystem                     
Redmine plugins:
  redmine_wysiwyg_editor         0.25.0
```

**Comment by taqueci** [comment]
Hi @mathieu-mbru and @nanego

I can not reproduce the issue with the latest version 0.25.0.
Can you please your Redmine environment, your iPhone information and steps to reproduce the issue?

**Comment by nanego** [comment]
Hi @taqueci 
I have just tested without any other plugin and the problem did not appear. It must come from a conflict with one of our plugins. I will identify which one and fix it. 
Thank you for your help.

**Comment by mathieu-mbru** [comment]
Hello
I still have the problem

![Iphone](https://user-images.githubusercontent.com/75627140/157423526-f25b87a2-bc85-4e79-a2e0-b5279c689d85.gif)
![image](https://user-images.githubusercontent.com/75627140/157423567-7d79f2dd-80b1-421d-a550-86c1507dadf3.png)

**Comment by taqueci** [comment]
Hi @mathieu-mbru 

Thank you for your explanation.
I understand.

The issue has been resolved by pull request #148.
(However, the issue still remains with Android.)
